static/app/views/settings/project/dynamicSampling/modals/recommendedStepsModal.spec.tsx

```diff
@@ -0,0 +1,180 @@
+import {renderGlobalModal, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+import {textWithMarkupMatcher} from 'sentry-test/utils';
+
+import {openModal} from 'sentry/actionCreators/modal';
+import {RecommendedStepsModal} from 'sentry/views/settings/project/dynamicSampling/modals/recommendedStepsModal';
+import {SERVER_SIDE_SAMPLING_DOC_LINK} from 'sentry/views/settings/project/dynamicSampling/utils';
+
+import {
+  getMockData,
+  mockedProjects,
+  mockedSamplingSdkVersions,
+  recommendedSdkUpgrades,
+  uniformRule,
+} from '../testUtils.spec';
+
+describe('Server-Side Sampling - Recommended Steps Modal', function () {
```

Comment:
Are we renaming to dynamic sampling here too or will that be a follow-up?

Reply:
I have updated it in my next commit... thanks!